src/components/Login.vue

```diff
@@ -17,39 +17,49 @@ const queryRedirect = z.object({
   }),
 });
 
+const working = ref(false);
+
 function login(event: Event) {
   // userStore.login(username.value)
   Promise.resolve()
   // Promise.reject('fake error')
       .then(() => {
+        working.value = true;
         error.value = '';
       })
       .then(userStore.login)
-      .then(() => {
-        try {
-          const maybeRedirect = queryRedirect.parse(route);
-          let redirect = maybeRedirect.query.redirect;
-          if (redirect.startsWith('/')) { // avoid possible third-party redirection
-            router.push(redirect);
-            return;
-          } else {
-            console.log('Refusing to redirect to', redirect);
-          }
-        } catch (e) {
-          // do nothing
-        }
-        router.push({
-          name: 'index'
-        });
-      })
+      // .then(() => {
+      //   try {
+      //     const maybeRedirect = queryRedirect.parse(route);
+      //     let redirect = maybeRedirect.query.redirect;
+      //     console.log('login component route,', route);
+      //     if (redirect.startsWith('/')) { // avoid possible third-party redirection
+      //       console.log('login component router push to redirect', redirect);
+      //       router.push(redirect);
+      //       return;
+      //     } else {
+      //       console.log('Refusing to redirect to', redirect);
+      //     }
+      //   } catch (e) {
+      //     // do nothing
+      //   }
+      //   console.log('login component redirect to index');
+      //   router.push({
+      //     name: 'index'
+      //   });
+      // })
       .catch(e => {
         console.debug('Error logging in', e);
         error.value = [
           'Error logging in.',
           'Ensure that your system has krb5 configured.',
           'Ensure that your browser has the correct trusted URIs for Negotiate authentication.',
+          'Ensure that you have logged into Kerberos on your system.',
```

Comment:
Should we make a separate help page related to these requirements for new members that can be accessed without logging in?

Reply:
Adding link to private kba in chat description in the future